Cars/Controllers/EmployeeRestController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cars.Repositories;
+using Cars.Model;
+using Cars.Dtos;
+using Cars.Extensions;
+using Cars.StaffDTO;
+
+namespace Cars.Controllers
+{
+    [ApiController]
+    [Route("staff")]
+    public class EmployeeRestController : ControllerBase
+    {
+        private readonly IEmployeeRepository repository;
```

Comment:
Controller doen't have repository at all. It just has a service. After calling method in service, the service calls method in Repo. Look how it works in for example Car Object.